contrib/Dockerfile.manylinux

```
@@ -97,6 +200,49 @@ ENV OPENSSL_ROOT_DIR="/usr/local/openssl3"
 ENV OPENSSL_LIBRARIES="/usr/local/openssl3/lib64:/usr/local/openssl3/lib"
 ENV OPENSSL_INCLUDE_DIR="/usr/local/openssl3/include"
 
+# --- INFINIA (libred_client) transitive deps not packaged at the right soname on EL8 ---
+# liburing 2.x: EL8 only ships liburing.so.1 (1.0.7); libred_client needs liburing.so.2.
+# Pinned to 2.14 to match subprojects/liburing.wrap (the version NIXL's own build uses); this
+# system install is found ahead of that meson wrap and also satisfies the prebuilt libred_client's
+# transitive liburing.so.2 link/runtime need (the wrap is an internal subproject not on the
+# system linker/loader path, so it can't resolve libred's NEEDED — hence the system build here).
+# Build/install ONLY the library (make -C src): liburing 2.14's examples (e.g. zcrx.c) pull in
+# linux/udmabuf.h, which isn't in the manylinux_2_28 (EL8) kernel headers, so a full `make` fails.
+RUN cd /tmp && \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)